python/lsst/pipe/tasks/subtractBrightStars.py

```diff
@@ -364,6 +372,32 @@ def run(
 
         return subtractorExp, invImages, badStamps
 
+    def detInRegions(self, inputs):
```

Comment:
Is this intended to be used globally, outside this file? If not, we might consider renaming this to `_detInRegions`.